nixos/modules/programs/firefox.nix

```
@@ -38,7 +38,7 @@ in
 
     wrapperConfig = mkOption {
       type = types.attrs;
-      default = {};
+      default = { };
```

```suggestion
      default = {};
```
I believe we try to avoid drive-by formatting changes in nixpkgs.